wicket-core/src/main/java/org/apache/wicket/ajax/res/js/wicket-event-jquery.js

```diff
@@ -277,6 +277,10 @@
 			* @param topic {String} - the channel name for which all subscribers will be notified.
 			*/
 			publish: function (topic) {
+				if (topic === Topic.AJAX_HANDLERS_BOUND) {
+            		console.warn("As of Wicket 8, Topic.AJAX_HANDLERS_BOUND has been deprecated in favour of Topic.AJAX_ONDOMREADY_EVENTS_BOUND and will be removed in Wicket 9.")
+            		topic = Topic.AJAX_ONDOMREADY_EVENTS_BOUND;
+            	}
```

Comment:
To be backward compatible we will have to still publish this event together with the new one.
This check and warning should be moved to `#subscribe()`.